src/components/Utilities/Button.tsx

```diff
@@ -1,5 +1,14 @@
-import { FC, PropsWithChildren } from 'react';
+import { FC, ReactNode } from 'react';
 
-const Button: FC<PropsWithChildren> = ({ children }) => <button type="button">{children}</button>;
+interface ButtonProps {
+  onClick?: any;
+  children: ReactNode;
+}
+
+const Button: FC<ButtonProps> = ({ onClick = {}, children }) => (
```

Comment:
onClick is defaulted to `{}`. It means if nothing is passed to onClick onClick would be `{}`, and then if we call it `({})()` if will throw an error